src/components/Community/Meet/index.js

```diff
@@ -110,7 +111,24 @@ CommunityIssue.propTypes = {
   color: PropTypes.string
 }
 
-export default function CommunityMeet({ issues, theme, topics }) {
+export default function CommunityMeet({ theme }) {
+  const [isIssuesLoaded, setIsIssuesLoaded] = useState(false)
+  const [issues, setIssues] = useState([])
+
+  const [isTopicsLoaded, setIsTopicsLoaded] = useState(false)
+  const [topics, setTopics] = useState([])
+
+  useEffect(() => {
+    getLatestIssues().then(result => {
+      setIssues(result)
+      setIsIssuesLoaded(true)
+    })
+    getLatestTopics().then(result => {
+      setTopics(result)
+      setIsTopicsLoaded(true)
+    })
```

Comment:
Nit: this doesn't stop the set calls when the component has already unmounted. It's not a big deal but it generates a warning in the console.

Reply:
Good observation. I will think about how to solve it, but maybe will leave it to the future.

Reply:
Usually I do something like this:

```
useEffect(() => {
  let canceled = false
  asyncStuff().then(() => {
    if (canceled) return
  })
  return () => {
    canceled = true
  }
}, [])
```

It works for mount/unmount as well as one of the useEffects dependencies changing.

Reply:
Good idea, I will do it this way!